experiments/tf_trainer/common/dataset_config.sh

```diff
@@ -23,6 +25,31 @@ elif [ "$1" == "many_communities" ]; then
     # removed is a boolean variable cast as an int.
     # 1 means that the comment was removed and 0 means it was not.
     label_dtypes="int"
+    text_feature="comment_text"
+
+elif [ "$1" == "many_communities_40_per_8_shot" ]; then
+
+    if [ "$2" == "optimistic" ]; then
+        train_path="${GCS_RESOURCES}/transfer_learning_data/many_communities_40_per_8_shot/augmented_train.tfrecord"
+    elif [ "$2" == "pessimistic" ]; then
+        train_path="${GCS_RESOURCES}/transfer_learning_data/many_communities_40_per_8_shot/original_train..tfrecord"
```

Comment:
Typo "original_train..tfrecord" -> "original_train.tfrecord"

Reply:
Oh wait, there must have been a typo in the file which wrote these tfrecords. Darn. Ok to merge this for now, when I update the filenames I'll change the code here.